Services/DataBaseService/Utils/MigrationScripts/24.03.2020/0002_AddIsExistColumnToUsersCredential.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE dbo.UsersCredentials ADD IsExist BIT DEFAULT 1;
```

Comment:
Again, I don't like the name

Reply:
Agree. Moreover, IsActive sounds better for me since we can Deactivate the user by deleting him, and then Activate via hotline's admin panel

Reply:
I have answered abow